lib/src/cmaf/manifest-samples/dash-samples/sample-1/manifest-sample-1.mpd

```diff
@@ -3,23 +3,23 @@
   <Period start="PT0S" duration="PT50S" id="1">
     <AdaptationSet mimeType="video/mp4" frameRate="30/1" segmentAlignment="true" subsegmentAlignment="true" startWithSAP="1" subsegmentStartsWithSAP="1" bitstreamSwitching="false">
       <SegmentList timescale="90000" duration="900000">
```

Comment:
The SegmentList was outside of Representation tag in the sample where you take it from? I'm asking because in the code I'm seeing that we covered the cases where the Segment tags are inside of a Representation tag. And to cover this new case we will have to add more cases in our code.

Reply:
If you take a look at the manifest2, the SegmentTemplate is also outside the Representation, though in the manifest4 it is inside.